base.rb

```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
```

Comment:
__[method_definition_validator]__
* Not enough arguments. Did you forget the following arguments? [p] at [base.rb#L14](/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/base.rb#L14)
* Not enough arguments. Did you forget the following arguments? [p] at [use.rb#L1](/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/use.rb#L1)